hack/run-e2e-test.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -uo pipefail
+
```

Comment:
The original commit had a `go get` here for ginkgo. But it was setting `GO111MODULE=on` and using `go get -u`. My original comment was to remove the env var settting and not use `-u`. But the `go get` command itself is still necessary.

For example, I just checked this commit out on my local machine, and made sure that I did not have `ginkgo` in `$GOPATH/bin`, and this happened:

```
[trhoden@mbp ~/git/sigs.k8s.io/vsphere-csi-driver ((2b2af64...))]$ hack/run-e2e-test.sh
hack/run-e2e-test.sh: line 28: ginkgo: command not found
```

We need to add that `go get` back so that the executable is installed when this test is run. It will install the version already listed in `go.mod`.

Reply:
I understand. I believe this error:
`ginkgo: command not found`
is a generic error with/without the `go get `
But I have added the change back as per your request and also mentioned here: [http://onsi.github.io/ginkgo/#the_ginkgo_cli](url)